parsl/executors/high_throughput/executor.py

```diff
@@ -545,10 +576,7 @@ def submit(self, func, resource_specification, *args, **kwargs):
         Returns:
               Future
         """
-        if resource_specification:
-            logger.error("Ignoring the call specification. "
-                         "Parsl call specification is not supported in HighThroughput Executor.")
-            raise UnsupportedFeatureError('resource specification', 'HighThroughput Executor', None)
+        validate_resource_spec(resource_specification)
```

Comment:
what happens when this returns false?

Reply:
`HTEX.submit` will fail with `InvalidResourceSpecification`, which I think is reasonable.